thirdparty/.gitignore

```diff
@@ -0,0 +1 @@
+angelscript
```

Comment:
angelscript is checked in so it should not be ignored completely! Perhaps you want to ignore `libangelscript.a`?